cinema/serializers.py

```diff
@@ -1 +1,66 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Actor, CinemaHall, Genre, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
```

Comment:
The `ActorSerializer` includes `full_name` in the fields list. Since `full_name` is a property and not a model field, you need to define it explicitly in the serializer using a `SerializerMethodField` or similar approach to ensure it can be serialized properly.